raiden/channel.py

```diff
@@ -508,7 +511,7 @@ def __init__(self, our_state, partner_state, external_state,
             # transfer to be included in a block. This is the time to close a
             # channel and then to unlock a lock on chain.
             #
-            raise ValueError('reveal_timeout must be at least 1')
```

Comment:
Ehhmm ... which is why I removed it, and changed it to 3. You are commenting on the red part :)